Remove SiteIsolationOverlay

achristensen07 opened the pull request:
#### 6c83992cd184cbe81c2a38372aa918f60b7a78ef
<pre>
Remove SiteIsolationOverlay
<a href="https://bugs.webkit.org/show_bug.cgi?id=297086">https://bugs.webkit.org/show_bug.cgi?id=297086</a>
<a href="https://rdar.apple.com/157796265">rdar://157796265</a>

Reviewed by Charlie Wolfe.

It only ever worked in the main frame process, wasn&apos;t terribly useful,
and has since been superseded by &quot;Show frame process borders&quot; which works
in all processes and gives a consistent visual indicator of site isolation.

* Source/WebCore/page/DebugPageOverlays.cpp:
(WebCore::SiteIsolationOverlay::drawRect):

Canonical link: <a href="https://commits.webkit.org/298415@main">https://commits.webkit.org/298415@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/286642883ac6763fb4f89d95de78061536f20966

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/115461 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/35151 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/25654 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/121544 "Built successfully") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/66032 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/b60c12dd-f50d-4269-903a-4e3d92bbe035) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/117350 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/35812 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/43722 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/87722 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/66032 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/9ba8d09a-7de2-41e2-98f3-dc46cee98352) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/118409 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/28564 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/103636 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/68114 "Passed tests") | | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/27722 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/21758 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/65201 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/97953 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/21872 "Passed tests") | [❌ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/124708 "Failed to checkout and rebase branch from PR 49091") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/42394 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/31767 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/2/builds/124708 "Failed to checkout and rebase branch from PR 49091") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/42761 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/99825 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/2/builds/124708 "Failed to checkout and rebase branch from PR 49091") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/41518 "Passed tests") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/19381 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/38290 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/18470 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/42275 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/47835 "Built successfully") | | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/125/builds/41776 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/45103 "Built successfully") | | | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/124/builds/43493 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/26b455697219de5275993bbd60f329872db551e4. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/49091

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/b1769d1bfde39d1c94586e494761906e238f46d1. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/49091

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/286642883ac6763fb4f89d95de78061536f20966. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/49091

achristensen07 committed:
Remove SiteIsolationOverlay
https://bugs.webkit.org/show_bug.cgi?id=297086
rdar://157796265

Reviewed by Charlie Wolfe.

It only ever worked in the main frame process, wasn't terribly useful,
and has since been superseded by "Show frame process borders" which works
in all processes and gives a consistent visual indicator of site isolation.

* Source/WebCore/page/DebugPageOverlays.cpp:
(WebCore::SiteIsolationOverlay::drawRect):

Canonical link: https://commits.webkit.org/298415@main
